Users/abehenderson/textfiles/log.txt

```diff
@@ -0,0 +1,4 @@
+I used mkdir Users/abehenderson/textfiles to create a new directory 'textfiles'.
```

Comment:
Let's make sure to use backticks when we write terminal commands, here and below.

Reply:
Sooo...
I used `mkdir Users/abehenderson/textfiles` to create a new directory 'textfiles'.

...?

Reply:
Yes